closeio_api/__init__.py

```diff
@@ -48,7 +48,8 @@ def __init__(self, base_url, api_key=None, tz_offset=None, max_retries=5,
 
         self.session.headers.update({
             'Content-Type': 'application/json',
-            'X-TZ-Offset': self.tz_offset
+            'X-TZ-Offset': self.tz_offset,
+            'User-Agent': 'close python %s' % requests.utils.default_user_agent()
```

Comment:
If we include this I suggest it be the actual name of the python package (`closeio`) and also include the version number.

Reply:
For now, I'll just get rid of the user agent change, and I'll address it in another PR so we can get this shipped out.